### PR TITLE
few improvements to allow fedora/el deployments without errors

### PR DESCRIPTION
**Pull request by mpolednik**

### CODE REVIEW

**Comment by rmohr** [comment]
LGTM waiting for @fabiand

File: README.md [hunk 1]
```diff
@@ -52,6 +52,12 @@ On Fedora these are provided by the following packages:
 $ dnf install -y make qemu-system-x86 libguestfs-tools-c expect
 ```
 
+QEMU has to be configured to run under the `root` user:
+
+```
+echo -e 'user = "root"\ngroup = "root"' >> /etc/libvirt/qemu.conf
```
Comment (rmohr) [inline README.md]: Would be good to clarify why it is needed in your setup.

**Comment by mpolednik** [comment]
Dropped the README change, could be (and most likely is) some problem with my setup.

**Comment by rmohr** [comment]
Thx. :cake

**Pull request merged**